helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -100,3 +112,33 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
         printStackTrace = stacktrace
     }
 }
+
+/**
+ * Read [ortFile] into an [OrtResult] and return it. Make sure that information about project scopes is available
+ * (by calling [OrtResult.withResolvedScopes]), so that it can be processed.
```

Comment:
Hmm, I actually do not see `withResolvedScopes` being called here...

Reply:
Yes, that sentence should have gone to the next commit.